cpp/unreal_projects/SpearSim/Config/DefaultGame.ini

```diff
@@ -8,6 +8,7 @@ ProjectID=51C548154BC1CB53E468FCA71A4A73EA
 
 [/Script/UnrealEd.ProjectPackagingSettings]
 bShareMaterialShaderCode=False ; we need this parameter to be False, otherwise Unreal will not find our materials
+bUseIoStore=False
```

Comment:
```
bShareMaterialShaderCode=False ; we need this parameter to be False, otherwise Unreal will not find our materials
bUseIoStore=False              ; use Unreal's legacy asset loading system for compatibility with our existing approach in build_paks.py
```

Reply:
Updated this